docs/configuration/monitoring.md

```diff
@@ -1,43 +1,140 @@
 # Monitoring
 
-In older versions, Codacy was monitored using an application we build
-in-house called "Crow". Going forward, we are deprecating Crow and
-setting up "Prometheus style" metrics in all our components that
-can be accessed using the Prometheus + Grafana ecosystem.
+Currently two monitoring solutions are supported:
 
-Initially Crow will still be bundled in Codacy helm chart, until
-we have all functionality available on Grafana.
+* [**Crow**](##Setting-up-monitoring-using-Crow) - A simple, lightweight, and builtin monitoring solution.
+* [**Prometheus+Grafana+Loki**](##Setting-up-monitoring-using-Grafana,-Prometheus,-and-Loki) - A more comprehensive third-party monitoring solution, for more advanced usage.
 
-## Setting up monitoring using Grafana & Prometheus
+## Setting up monitoring using Crow
+
+Crow is a visualization tool that displays information about the projects and jobs
+that are pending for analysis, as well as the ones running in the Codacy platform.
+
+The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
+It can be accessed through the `/monitoring` path of the url pointing to your Codacy
+installation, e.g. `http://codacy.company.org/monitoring`. You must set the
+`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
+projects can be properly established inside `crow`.
+
+For example:
+
+```yaml
+global:
+  codacy:
+    crow:
+      url: "http://codacy.example.com/monitoring"
+```
+
+Please see the [README.md](https://github.com/codacy/chart/blob/master/README.md) for more information about these values.
+
+### Configuring your credentials
+
+You **should** provide a password for the `crow` installation either through the `values.yaml` file or through a `--set` parameter during the `helm` installation process. This parameter can be configured as follows:
```

Comment:
Defining the custom password is not mandatory (since there is a default one) but highly recommended, is this right? Also, to improve the documentation readability we can simply use the capitalized proper nouns of the tools throughout this page, when appropriate of course:

```suggestion
We highly recommend that you define a custom password for the Crow installation. You can do this either through the `values.yaml` file or through a `--set` parameter during the Helm installation process. This parameter can be configured as follows:
```

Reply:
"Should" is generally used to convey a recommendation or an expected behaviour, and doesn't imply something is mandatory.